spdx/v2/v2_2/json/json_test.go

```diff
@@ -200,7 +172,136 @@ func Test_ShorthandFields(t *testing.T) {
 				Relationship: common.TypeRelationshipContains,
 			},
 		},
-	}, doc)
+	}
+
+	if diff := cmp.Diff(want, doc, cmpopts.IgnoreUnexported(spdx.Package{}), cmpopts.SortSlices(relationshipLess)); len(diff) > 0 {
+		t.Errorf("got incorrect struct after parsing JSON example: %s", cmp.Diff(want, doc, cmpopts.IgnoreUnexported(spdx.Package{})))
```

Comment:
`cmp.Diff(want, doc, cmpopts.IgnoreUnexported(spdx.Package{}))` should just be `diff` here, right?